client/components/cloud/project/billing/translations/Messages_fr_FR.json

```diff
@@ -57,5 +57,11 @@
   "cpbc_server_details_popover_image_type": "Type",
   "cpbc_server_details_popover_vm_type": "Type de serveur",
   "cpbc_server_details_popover_imagetype_linux": "Linux",
-  "cpbc_server_details_popover_imagetype_windows": "Windows"
```

Comment:
Doesn't seems to be a feature but a fix I guess.

According to the fact that you've just added missing translations.